src/main/target/OMNIBUSF4/target.h

```diff
@@ -172,10 +172,17 @@
 #define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
 #define FLASH_CS_PIN            SPI3_NSS_PIN
 #define FLASH_SPI_INSTANCE      SPI3
-#define USE_FLASHFS
-#define USE_FLASH_M25P16
 #endif // OMNIBUSF4
 
+#define USE_RX_SPI
```

Comment:
Probably better to add this inside a conditional, so that it only gets built for OMNIBUSF4, but not for the variants.

Reply:
Agreed, right now most of the variants fail because of this. Fixing...